hypervisor/qemu/qemu_process.go

```diff
@@ -13,6 +14,47 @@ import (
 	"github.com/hyperhq/runv/hypervisor"
 )
 
+// implement io.Reader interface
+type qemuLogReader struct {
+	logFile string
+	offset  int64
+}
+
+func (f *qemuLogReader) Read(p []byte) (n int, err error) {
+	reader, err := os.Open(f.logFile)
+	defer reader.Close()
+	if err != nil {
+		return 0, err
+	}
+	reader.Seek(f.offset, 0)
+
+	n, err = reader.Read(p)
+
+	if err == io.EOF {
+		time.Sleep(1 * time.Second)
+	}
+	f.offset += int64(n)
+
+	return n, err
+}
+
+func watchLogFile(logFile string) {
+	file := &qemuLogReader{logFile, 0}
+	br := bufio.NewReader(file)
+
+	for {
+		log, _, err := br.ReadLine()
+		if err == io.EOF {
```

Comment:
After implementing createQemuLogReader , the error of io.EOF would be the sign of return from wathLogFile.